docs/mp/integrations/jedis.adoc

```diff
@@ -62,19 +74,26 @@ The following examples show how to create and inject a Jedis pool named `orders`
  }
 ----
 
-The extension implements this injection point by creating a
- link:{jedis-base-url}/jedis/JedisPool.html[JedisPool] object in the
- link:{jakarta-cdi-javadoc-url}/javax/enterprise/context/ApplicationScoped.html[application scope].
+Helidon implements this injection point by creating a
+link:{jedis-base-url}/jedis/JedisPool.html[JedisPool] object in the
+link:{jakarta-cdi-javadoc-url}/jakarta/enterprise/context/ApplicationScoped.html[application scope].
 
 You can configure the object using xref:../jaxrs/server-configuration.adoc[MicroProfile config].
- For example, the Jedis pool created above can be configured as follows:
+For example, the Jedis pool created above can be configured as follows:
```

Comment:
These two lines are duplicate i.e. its in configuration section below too.

Reply:
fixed